app/src/main/java/org/mozilla/fenix/compose/ClickableSubstringLink.kt

```diff
@@ -86,7 +87,7 @@ fun ClickableSubstringLink(
 private fun ClickableSubstringTextPreview() {
     val text = "This text contains a link"
 
-    FirefoxTheme {
+    FirefoxTheme(theme = Theme.getTheme(isPrivate = false)) {
```

Comment:
Would it make sense to create a separate preview theme, if only to save some characters when creating previews? Something like:

```
@Composable
fun PreviewTheme(isPrivate: Boolean = false, content: @Composable () -> Unit) {
    FirefoxTheme(theme = Theme.getTheme(isPrivate = isPrivate), content = content)
}
```

Reply:
See https://github.com/mozilla-mobile/fenix/pull/24527#discussion_r838957235. In general, I think what you're proposing is right. The ideal solution would be to avoid having the need for either the `PreviewTheme` or `isPrivate` parameter and for `FirefoxTheme` to know it is dealing with a @Preview, but we're stuck with this until Compose 1.1.0. I don't know if it's worth the effort knowing that both of these solution would hopefully be temporary.